libbeat/common/dtfmt/ctx.go

```diff
@@ -31,7 +31,7 @@ type ctx struct {
 	isoWeek, isoYear int
 
 	hour, min, sec int
-	millis         int
+	nano           int
```

Comment:
Should this explicitly be an `int64`? I'm not sure we run on 32-bit system where that might matter though.

Reply:
`time.Time.Nanosecond` returns an `int` in the range [0, 999999999], so this is safe.